apps/src/AzureTextToSpeech.js

```diff
@@ -1,9 +1,17 @@
 import {hashString, findProfanity} from '@cdo/apps/utils';
+import Sounds from '@cdo/apps/Sounds';
 
 // XMLHttpRequest readyState 4 means the request is done.
 // https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/readyState
 const READY_STATE_DONE = 4;
 
+/**
+ * A packaged response for a requested sound. Used for caching and for playing sound bytes.
```

Comment:
bites? :)

Reply:
lol should it be? i was thinking of it as "bytes that represent a sound" but could definitely also be called a "sound bite"

Reply:
Hah, if it's deliberate than I'm a fan!